src/tasks/task1.js

```diff
@@ -0,0 +1,25 @@
+function sayHello(name) {
+   if (typeof name !== 'string')
+      return

```

Comment:
if you are going to use if statement in one line, you must write it lie `if (typeof name !== 'string') return;`